lesson06/index.js

```diff
@@ -0,0 +1,30 @@
+const attributes = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const ul = document.getElementById("js-lists");
+
+const fragment = document.createDocumentFragment();
+
+const promise = new Promise(resolve => {
+    setTimeout(() => resolve(attributes), 3000);
+});
+
+promise.then(attributes => {
+    attributes.forEach(attribute => {
```

Comment:
IMO
As for the `attributes` specified as an argument here, I thought it would be better to avoid using the same name as a constant that has already been defined.
```js
const attributes = [
    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
];
```
I would use `values`, `listDatas`, etc. These are not the correct answers, but I think it is better to use different names to avoid confusion later on.

However, this one is IMO, so I'll leave it up to you to decide if you want to fix it or not. The code is Approve!

Reply:
@443n0511 
IMO
The plural of data is data. maybe

Reply:
@kenmori 
Thank you very much. How about something like this? I want to improve my English ...😇
```suggestion
promise.then(listData => {
    listData.forEach(data => {
```

Reply:
@443n0511
Thank you very much for the review!

MDN says "then() method returns a new Promise object completely different from the first Promise
and returns a new Promise object that is completely different from the first one" !

I thought it should be a different argument name so that it can be understood.
I did not understand it properly. Thank you very much!!